build/visible_to/BUILD

```diff
@@ -331,6 +331,13 @@ package_group(
     ],
 )
 
+package_group(
```

Comment:
do you want to restrict visibility or were you trying to fix the build?

Reply:
wanted to fix the build.